src/RoleObjects/ArticleData.test.js

```diff
@@ -0,0 +1,80 @@
+import * as createjs from 'createjs-module';
+import AccessibilityModule from '../index';
+
+describe('ArticleData', () => {
+  describe('register role', () => {
+    let canvasEl;
+    let parentEl;
+    let stage;
+    let container;
+    let cjsArticle;
+    let articleEl;
+    let positionVal;
+    let sizeVal;
+
+    beforeEach(() => {
+      canvasEl = document.createElement('canvas');
+      parentEl = document.createElement('div');
+      stage = new createjs.Stage(canvasEl);
+      container = new createjs.Container();
+      cjsArticle = new createjs.Shape(); // dummy object
+      positionVal = 7;
+      sizeVal = 99;
+
+      AccessibilityModule.register({
+        displayObject: container,
+        role: AccessibilityModule.ROLES.MAIN,
+      });
+      AccessibilityModule.setupStage(stage, parentEl);
+      stage.accessibilityTranslator.root = container;
+      stage.addChild(container);
```

Comment:
- would it make sense to move the logic of setting up the stage and main container to an imported/reusable helper function?

Reply:
Eh, depends on how many LOC we're talking about. There's also an `onReady` that you might want to use (maybe not here, but centralizing just a couple of setup lines might make that a bit harder to use).

Reply:
- added helper file